src/cli/project/version/bump.rs

```diff
@@ -0,0 +1,36 @@
+use crate::Project;
+use rattler_conda_types::VersionBumpType;
+
+pub async fn execute(mut project: Project, bump_type: VersionBumpType) -> miette::Result<()> {
+    // get version and exit with error if not found
+    let current_version = project
+        .version()
+        .as_ref()
+        .ok_or_else(|| {
```

Comment:
I think you can use a miette::miette! here and error out  instead of unwrapping and printing.

Reply:
I replaced it by `.ok_or_else(|| miette::miette!("No version found in manifest."))?` Is that ok or were you asking for something else than `ok_or_else`?